pontoon/base/migrations/0129_translation_active.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-29 10:58
+from __future__ import unicode_literals
+
+from django.db import connection, migrations, models
+from django.db.utils import ProgrammingError
+
+entity_document_update_trigger_drop_sql = '''
+    DROP TRIGGER base_translation_entity_document_update ON "base_translation";
+    DROP FUNCTION base_translation_entity_document_update();
+'''
+
+
+def drop_entity_document(apps, schema):
+    with connection.cursor() as cursor:
+        try:
+            cursor.execute(entity_document_update_trigger_drop_sql)
+        except ProgrammingError:
+            pass
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('base', '0128_pontoon-intro_to_system_project'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='translation',
+            name='entity_document',
+        ),
+        migrations.AddField(
+            model_name='translation',
+            name='active',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.RunPython(
```

Comment:
[nit] you can use `migrations.RunSQL` and `migrations.RunSQL.noop` respectively

Reply:
That was the initial implementation, but then it resulted in `ProgrammingError` (which is now catched) as I was applying and unapplying migrations during development.